apps/supporting-documents/acceptance/features/_happy-path.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+const baseUrl = require('../../').baseUrl || '';
+
+// create a function that returns the next item of an array every time it is called
+const iterate = (arr) => {
+  let count = 0;
+  return () => {
+    const value = arr[count];
+    count++;
+    return value || arr[arr.length - 1];
+  };
+};
```

Comment:
You've seen it before, that's why.

I've tried thinking of a good place for it to move it, without joy. Mostly because I'd like to make it automatic behaviour in the auto-filler, but can't come up with a nice way of handling it that is safely different from using arrays to specify multiple checkbox selections.

While I ponder on a *good* abstraction I figure no abstraction is better than a bad one. It's not exactly a huge amount of code.